dca/tl_nc_queue.php

```diff
@@ -0,0 +1,157 @@
+ * Software Foundation website at <http://www.gnu.org/licenses/>.
+ *
+ * PHP version 5
+ * @copyright  terminal42 gmbh 2014
+ * @license    LGPL
+ */
+
+/**
+ * Table tl_nc_queue
+ */
+$GLOBALS['TL_DCA']['tl_nc_queue'] = array
+(
+
+    // Config
+    'config' => array
+    (
+        'dataContainer'               => 'Table',
+        'closed'                      => true,
+        'notEditable'                 => true,
+        'notCopyable'                 => true,
+        'notSortable'                 => true,
+        'sql' => array
+        (
+            'keys' => array
+            (
+                'id'            => 'primary',
+                'message'       => 'index',
+                'sourceQueue'   => 'index',
+                'targetGateway' => 'index',
+            )
+        )
+    ),
+
+    // List
+    'list' => array
+    (
+        'sorting' => array
+        (
+            'mode'                    => 2,
+            'fields'                  => array('dateAdded DESC', 'id DESC'),
+            'panelLayout'             => 'filter;search,limit'
+        ),
+        'label' => array
+        (
+            'fields'                  => array('message', 'dateSent'),
+            'label_callback'          => array('NotificationCenter\tl_nc_queue', 'listRows'),
+        ),
+        'operations' => array
+        (
+            're-queue' => array
+            (
+                'label'               => &$GLOBALS['TL_LANG']['tl_nc_queue']['re-queue'],
+                'href'                => 'key=re-queue',
+                'icon'                => 'system/modules/notification_center/assets/re-queue.png',
+                'attributes'          => 'onclick="if(!confirm(\'' . $GLOBALS['TL_LANG']['tl_nc_queue']['re-queueConfirmation'] . '\'))return false;Backend.getScrollOffset()"',
+                'button_callback'     => array('NotificationCenter\tl_nc_queue', 'reQueueButton')
+            ),
+            'delete' => array
+            (
+                'label'               => &$GLOBALS['TL_LANG']['tl_nc_queue']['delete'],
+                'href'                => 'act=delete',
+                'icon'                => 'delete.gif',
+                'attributes'          => 'onclick="if(!confirm(\'' . $GLOBALS['TL_LANG']['MSC']['deleteConfirm'] . '\'))return false;Backend.getScrollOffset()"',
+                'button_callback'     => array('NotificationCenter\tl_nc_queue', 'deleteButton')
+            ),
+            // @todo: maybe format the json encoded tokens for better usability?
+            'show' => array
+            (
+                'label'               => &$GLOBALS['TL_LANG']['tl_nc_queue']['show'],
+                'href'                => 'act=show',
+                'icon'                => 'show.gif'
+            )
+        )
+    ),
+
+    // Palettes
+    'palettes' => array
+    (
+        'default'                     => '{title_legend},title,type'
```

Comment:
According to me the array does not need to be there if the records won't be altered through a regular back end options.